docs/src/other.md

```diff
@@ -171,3 +171,12 @@ true
 ```@docs
 show_gap_help
 ```
+
+## Objects that must be documented, otherwise Documenter.jl throws an error.
```

Comment:
If we include these, then I think we should follow the example Documenter itself: it has a section "Developers" in its manual, and under it "Contributing" and "Internals".

Listing these under "Developers >> Internal" seems OK to me...

Reply:
O.k.
(The hierarchy of the Documenter.jl manual is deeper than that of the GAP.jl manual, thus the result looks different.)